EPiServer.VueStorefrontApiBridge/VueStorefrontApiBridgeRegisterEx.cs

```diff
@@ -32,10 +35,13 @@ public void ConfigureContainer(ServiceConfigurationContext context)
             if (!context.Services.Contains(typeof(IUserManager)))
                 context.Services.Add(typeof(IUserManager), typeof(DefaultUserManager), ServiceInstanceScope.Transient);
 
+            if (!context.Services.Contains(typeof(ICustomerContactManager)))
```

Comment:
I think this may cause some problems. What if someone will register his services after this code executes ? I think that for the time being its fine, however we will have to change this in the future.